docs/source/upcoming_release_notes/issue_number-add_ItechRfof.rst

```diff
@@ -0,0 +1,32 @@
+issue_number add ItechRfof
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- Instrumentation Technologies RF over Fiber unit
```

Comment:
I suggest including a class name to make it clear what was added. This becomes part of the release notes so it needs to make sense in context of being next to other bullet points.
```suggestion
- Added ItechRfofBase: Instrumentation Technologies RF over Fiber unit
```